mimic/session.py

```diff
@@ -84,7 +88,7 @@ def _new_session(self, username_key=None, **attributes):
                  with this :obj:`MimicCore`).
         :rtype: :obj:`Session`
         """
-        for key in ['username', 'token', 'tenant_id']:
+        for key in ['username', 'token', 'tenant_id', 'impersonated_token']:
```

Comment:
By having `impersonated_token` be a mappable attribute of the session like this, that means that a session has *one and only one* impersonated token.  This is wrong, because multiple admins might impersonate a given user; each impersonation is its own session, with its own token.

That's why the "subsession" was there before.  An impersonation session should share the `api_objects` attribute (i.e. "the tenant's actual stuff"), but should not share the metadata about the session (the token, the expiration time).